server/organization/views.py

```diff
@@ -42,23 +44,51 @@ class ModifyOrganizationInventoryItemsDataUpdate(generics.GenericAPIView):
     the Inventory Data is refreshed
     """
 
+    default_refresh_time = 15
+
     # Note: if other methods are added here, keep in mind that the permissions will need to change
     def get_permissions(self):
         permission_classes = PermissionFactory(self.request).get_general_permissions([])
         return [permission() for permission in permission_classes]
 
     def post(self, request):
         data = request.data
-        org_id = data.get('organization', '')
-        new_job_timing = int(data.get('new_job_timing', ''))
+
+        org_id = data.get('organization_id')
+        new_job_timing = int(data.get('time'))
 
         try:
             organization = Organization.objects.get(org_id=org_id)
+            if not new_job_timing:
+                new_job_timing = self.default_refresh_time
```

Comment:
what if we don't submit a time in the front end? Does this endpoint still get called?